RNIronSourceInterstitials.js

```diff
@@ -0,0 +1,57 @@
+import { NativeModules, NativeEventEmitter } from 'react-native';
+
+const RNIronSourceInterstitials = NativeModules.RNIronSourceInterstitials;
+const IronSourceInterstialsEventEmitter = new NativeEventEmitter(RNIronSourceInterstitials);
+
+const eventHandlers = {
```

Comment:
Instead of manually creating all of the maps here, let's just have an Array of allowed event names to avoid redundancy:
```
const eventNames = [
  'interstitialDidLoad',
  'interstitalDidShow',
  ...
];
```

Then you can create the map object underneath it like so:
```
const eventHandlers = eventNames.reduce((result, eventName) => {
  result[eventName] = new Map();
  return result;
}, {});
```
In the future, you can add new events just by appending a string in one location.